central/blob/datastore/datastore.go

```diff
@@ -71,3 +80,18 @@ func (d *datastoreImpl) GetBlobWithDataInBuffer(ctx context.Context, name string
 	}
 	return buf, blob, exists, err
 }
+
+// Search blobs
+func (d *datastoreImpl) Search(ctx context.Context, query *v1.Query) ([]pkgSearch.Result, error) {
+	return d.searcher.Search(ctx, query)
+}
+
+// SearchIDs searches and return blob IDs
+func (d *datastoreImpl) SearchIDs(ctx context.Context, q *v1.Query) ([]string, error) {
+	return d.searcher.SearchIDs(ctx, q)
+}
+
+// SearchBlobsWithoutData searches and return blob metadata only
+func (d *datastoreImpl) SearchBlobsWithoutData(ctx context.Context, q *v1.Query) ([]*storage.Blob, error) {
```

Comment:
What is the use case for this one?

Reply:
We got use case to get metadata only. This is the search version for such a case. It could be used to construct a manifest for example.

Reply:
`SearchBlobMetadata` might be more clear.  IDK.